python/lsst/obs/cfht/ingest.py

```diff
@@ -46,47 +45,13 @@
            }
 
 
+@deprecated(reason="MegaPrime no longer requires a specialist gen3 ingest task. Please use the default."
+            " Will be removed after v22.",
+            category=FutureWarning,
+            version="21.0")
```

Comment:
We've already released version 21, so `version` should be `"v22.0"`. That means the "will be removed" needs to be "after v23", barring special approval.

Reply:
I didn't want it hanging around for a year when no-one is using it anyhow but I take the point.